com.microsoft.mrtk.core/Interactors/IVariableReticle.cs

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using UnityEngine;
+using UnityEngine.XR.Interaction.Toolkit;
+
+namespace Microsoft.MixedReality.Toolkit
+{
+    /// <summary>
+    /// A reticle that implements some visual effect controllable by a single float value.
+    /// </summary>
+    public interface IVariableReticle
+    {
+        /// <summary>
+        /// Updates visuals as needed for the variable reticle.
+        /// </summary>
+        public void UpdateVisuals(VariableReticleArgs args);
+    }
+
+    public struct VariableReticleArgs
```

Comment:
Please add a summary block describing this structure and where it is used.  Use "see" tag too...   Something like


        /// <summary>
        /// Arguments used during a calls <see cref="IVariableReticle.UpdateVisuals "/>.
        /// </summary>